packages/env/lib/build-docker-compose-config.js

```diff
@@ -32,7 +32,7 @@ function getMounts( config, wordpressDefault = 'wordpress' ) {
 
 	const pluginMounts = config.pluginSources.map(
 		( source ) =>
-			`${ source.path }:/var/www/html/wp-content/plugins/${ source.basename }`
+			`${ source.path }:/var/www/html/wp-content/plugins/${ source.basename }:delegated`
 	);
 
 	const themeMounts = config.themeSources.map(
```

Comment:
Should we also add `delegated` to core source, mappings, and themes?

Reply:
[Theme](https://github.com/WordPress/gutenberg/blob/47d3df9961f309e51014ab7602968c2c07b35229/packages/env/lib/build-docker-compose-config.js#L40)
[Directory](https://github.com/WordPress/gutenberg/blob/47d3df9961f309e51014ab7602968c2c07b35229/packages/env/lib/build-docker-compose-config.js#L30)
[Core](https://github.com/WordPress/gutenberg/blob/47d3df9961f309e51014ab7602968c2c07b35229/packages/env/lib/build-docker-compose-config.js#L45)

Although YMMV. Delegated benefits prioritizing the container doing things, rather than the outside world knowing about each incremental change as they are happening. This likely needs more testing, but is the approach I've seen other users merge to solve their Mac + Docker issues.

If you ran with a single WordPress and theme, delegated may not for instance be necesarry as there are less write operations, meaning it should be less likely to cause significant overhead communicating changes that may happen a lot.

One other thing we never got enough time to focus on was making write-heavy changes in a /tmp or similarly memory backed location (webpack-dev-server running in RAM and being 10x faster was what spurred that line of enquiry).